src/files/BrsFile.ts

```diff
@@ -188,13 +204,6 @@ export class BrsFile {
         //event that fires anytime a dependency changes
         this.unsubscribeFromDependencyGraph = this.program.dependencyGraph.onchange(this.dependencyGraphKey, () => {
             this.resolveTypdef();
-
-            //if there is no typedef file, and this file hasn't been parsed yet, parse it now
-            //(solves issue when typedef gets deleted and this file had skipped parsing)
-            if (!this.hasTypedef && this.wasParseSkipped) {
```

Comment:
This is no longer necessary, since the parser is a getter, it will parse the file on-demand the first time it's needed.